enos/ansible/roles/init_os/templates/init.sh.j2

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+
+set -x
+
+DNS_NAMESERVER={{ provider_net.dns }}
+SUBNET_RANGE={{ provider_net.cidr }}
+ALLOCATION_POOL_START={{ provider_net.start }}
+ALLOCATION_POOL_END={{ provider_net.end }}
+GATEWAY={{ provider_net.gateway }}
+
+
```

Comment:
add images to glance: https://github.com/BeyondTheClouds/enos/blob/master/enos/enos.py#L254-L261